packages/react-ui/package.json

```diff
@@ -56,6 +56,7 @@
     "@svgr/core": "^6.0.0",
     "@testing-library/react": "^12.0.0",
     "@testing-library/react-hooks": "7.0.2",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
je suis pas arrivé à ecrire un failing test sans user event (on a besoin de taper les characters). Dans la doc ils disent ça :
"Most projects have a few use cases for fireEvent, but the majority of the time you should probably use [@testing-library/user-event](https://testing-library.com/docs/user-event/intro)."